esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -1823,25 +1835,96 @@ int EspHttpBinding::onGetVersion(IEspContext &context, CHttpRequest* request, CH
     StringBuffer srvQName;
     qualifyServiceName(context, service, NULL, srvQName, NULL);
     verxml.appendf("<VersionInfo><Service>%s</Service><Version>%.3f</Version></VersionInfo>", srvQName.str(), m_wsdlVer);
-    
+
     response->setContent(verxml.str());
     response->setContentType(HTTP_TYPE_APPLICATION_XML_UTF8);
     response->setStatus(HTTP_STATUS_OK);
     response->send();
     return 0;
 }
 
+void getServiceSchema(const char *serviceQName, const char *methodQName, double version, const char *nstr,
+                      const StringBuffer &xmlFilename,  Owned<IProperties> &params, EsdlXslTypeId EsdlXslType,
+                      IProperties *opts, StringBuffer &schema)
+{
+    Owned<IEsdlDefinition> esdlDef;
+    Owned<IEsdlDefinitionHelper> defHelper;
+
+    esdlDef.set(createEsdlDefinition(nullptr, nullptr));
```

Comment:
These look like they would leak to me.  Have you tested with valgrind?  To hand ownership to "Owned" you should use setown().
But the easiest thing would be just to do:

```
Owned<IEsdlDefinition> esdlDef = createEsdlDefinition(nullptr, nullptr);
Owned<IEsdlDefinitionHelper> defHelper = createEsdlDefinitionHelper();

```